Dockerfile_archive

```diff
@@ -1,4 +1,17 @@
-FROM docker.io/debian:testing-slim AS build
+# Pull (Pre-Built):
+#   $ docker pull ghcr.io/jelmer/janitor/archive:latest
+# Build:
+#   $ docker build -t ghcr.io/jelmer/janitor/archive:latest -f Dockerfile_archive .
+#   $ buildah build -t ghcr.io/jelmer/janitor/archive:latest -f Dockerfile_archive .
+# Run:
+#   $ docker run --rm --publish 9914:9914 --name janitor-archive ghcr.io/jelmer/janitor/archive:latest
+# Troubleshoot:
+#   $ docker run -it --rm -p 9914:9914 --entrypoint=/bin/bash ghcr.io/jelmer/janitor/archive:latest
+#   $ docker run --interactive --tty --rm --publish 9914:9914 --entrypoint=/bin/bash --volume $( pwd ):/mnt/janitor --workdir /mnt/janitor ghcr.io/jelmer/janitor/archive:latest
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+# https://hub.docker.com/_/rust
+FROM docker.io/rust:slim-bookworm AS build
```

Comment:
Why switch images here, rather than just using debian: ?

Reply:
Due to https://github.com/jelmer/janitor/issues/931#issuecomment-2661150602